actionners/kubernetes/sysdig/sysdig.go

```diff
@@ -0,0 +1,292 @@
+	k8sChecks "github.com/falcosecurity/falco-talon/internal/kubernetes/checks"
+	k8s "github.com/falcosecurity/falco-talon/internal/kubernetes/client"
+	"github.com/falcosecurity/falco-talon/internal/models"
+	"github.com/falcosecurity/falco-talon/internal/rules"
+	"github.com/falcosecurity/falco-talon/utils"
+)
+
+const (
+	Name          string = "sysdig"
+	Category      string = "kubernetes"
+	Description   string = "Capture the syscalls packets in a pod"
+	Source        string = "syscalls, k8saudit"
+	Continue      bool   = false
+	UseContext    bool   = false
+	AllowOutput   bool   = false
+	RequireOutput bool   = true
+	Permissions   string = `apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: falco-talon
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+- apiGroups:
+  - ""
+  resources:
+  - pods/exec
+  verbs:
+  - get
+  - create
+- apiGroups:
+  - "batch"
+  resources:
+  - jobs
+  verbs:
+  - get
+  - list
+  - create
+`
+	Example string = `- action: Create a syscall capture from a pod
+  actionner: kubernetes:sysdig
+  parameters:
+    duration: 10
+    buffer_size: 1024
+  output:
+    target: aws:s3
+    parameters:
+      bucket: my-bucket
+      prefix: /captures/
+`
+)
+
+var (
+	RequiredOutputFields = []string{"k8s.ns.name, k8s.pod.name", "ka.target.namespace, (ka.target.pod.name or ka.target.name)"}
+)
+
+type Parameters struct {
+	Image      string `mapstructure:"image"`
+	Scope      string `mapstructure:"scope" validate:"oneof=pod node"`
+	Duration   int    `mapstructure:"duration" validate:"gte=0,lte=30"`
+	BufferSize int    `mapstructure:"buffer_size" validate:"gte=128"`
+}
+
+const (
+	baseName     string = "falco-talon-sysdig-"
+	defaultImage string = "issif/sysdig:latest"
+	// defaultImage       string = "sysdig/sysdig:latest"
+	defaultScope       string = "pod"
+	defaultTTL         int    = 60
+	defaultDuration    int    = 5
+	defaultMaxDuration int    = 30
+	defaultBufferSize  int    = 2048
+)
+
+type Actionner struct{}
+
+func Register() *Actionner {
+	return new(Actionner)
+}
+
+func (a Actionner) Init() error {
+	return k8s.Init()
+}
+
+func (a Actionner) Information() models.Information {
+	return models.Information{
+		Name:                 Name,
+		FullName:             Category + ":" + Name,
+		Category:             Category,
+		Description:          Description,
+		Source:               Source,
+		RequiredOutputFields: RequiredOutputFields,
+		Permissions:          Permissions,
+		Example:              Example,
+		Continue:             Continue,
+		AllowOutput:          AllowOutput,
+		RequireOutput:        RequireOutput,
+	}
+}
+
+func (a Actionner) Parameters() models.Parameters {
+	return Parameters{
+		Duration:   defaultDuration,
+		Scope:      defaultScope,
+		BufferSize: defaultBufferSize,
+		Image:      defaultImage,
+	}
+}
+
+func (a Actionner) Checks(event *events.Event, _ *rules.Action) error {
+	if event.Source == "k8s_audit" {
```

Comment:
Why this check only present in this actionner?

Reply:
because capturing the syscalls are relevant when an exec is detected, I don't see other use cases where it could, this is the first actionner which can work with 2 sources